lib/clashom.gi

```diff
@@ -3061,6 +3061,8 @@ local cl;
   cl:=ConjugacyClassesForSmallGroup(G);
   if cl<>fail then
     return cl;
+  elif IsSolvableGroup( G ) and CanEasilyComputePcgs(G) then
```

Comment:
I added the "raw" `IsSolvableGroup` check here because (a) it is reasonably efficient for permutation groups, and its result can be useful in many subsequent computations, too, and (b) I felt that the existing check for `IsSimpleGroup` later on is not really cheaper either. But of course I might be mistaken -- but I rely on @hulpke to point that out  :-)

Reply:
For permutation groups `IsSimpleGroup` is not as expensive as it might seem, as it uses tools from the composition series mechanism. In most cases order alone bails out quickly.
IsSolvableGroup should be cheap enough, as the code running afterwards effectively would discover that the group is solvable.